Students/Stoica S. Razvan Cosmin/Assignment1/assign3/src/assign3/assign3.java

```diff
@@ -0,0 +1,38 @@
+package assign3;
+
+public class assign3
```

Comment:
According to [this](http://www.oracle.com/technetwork/java/codeconventions-135099.html) class name should have the first letter uppercase